arboretum/ibm_cloud/fetchers/fetch_cluster_list.py

```diff
@@ -0,0 +1,70 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IKS cluster list fetcher."""
+
+import json
+
+from arboretum.common.errors import CommandExecutionError
+from arboretum.common.utils import run_command
+
+from compliance.evidence import store_raw_evidence
+from compliance.fetch import ComplianceFetcher
+
+
+class ClusterListFetcher(ComplianceFetcher):
+    """Fetch the list of IBM Cloud clusters."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Initialize the fetcher object with configuration settings."""
+        cls.logger = cls.locker.logger.getChild(
+            'ibm_cloud.cluster_list_fetcher'
+        )
+        return cls
+
+    @store_raw_evidence('ibm_cloud/cluster_list.json')
+    def fetch_cluster_list(self):
+        """Fetch IBM Cloud cluster list."""
+        for account in self.config.get(
+                'org.ibm_cloud.cluster_list.config.account'):
```

Comment:
```python
        accounts = self.config.get('org.ibm_cloud.cluster_list.config.account')
        for account in accounts:
```

Reply:
applied this, thank you.
(BTW I remove `config` from the key name (`'org.ibm_cloud.cluster_list.config.account'` -> `'org.ibm_cloud.cluster_list.account'`) because it is obviously config without saying config...)